seed/seed.go

```diff
@@ -92,7 +93,13 @@ type Seed interface {
 }
 
 // Open returns a Seed implementation for the seed at seedDir.
-// TODO: more parameters for the Core20 case
-func Open(seedDir string) (Seed, error) {
+// label if not empty is used to identify a Core 20 recovery system seed.
+func Open(seedDir, label string) (Seed, error) {
```

Comment:
wouldn't it be better to have an OpenWithLabel or something? the "" at the caller seems rather obscure

Reply:
It's still a bit unclear what we exactly want in the end, see the TODO inside, I think it's likely that in the end this will take some kind of options struct.

Reply:
@chipaca anyway in the interim the rename seems a good idea, but after I am through my current chain of PRs